### DISCUSSION TITLE
How to play thru plugged-in earphones & Bose speakers ?

### DISCUSSION THREAD

**Discussion by vmars316** [smplayer-dev/smplayer#1084] [Q&A]
How to play thru plugged-in earphones and/or Bose speakers ? 
Plays only thru Laptop speakers .
Thanks for your Help...

**Comment by vmars316** [comment]
OOPS,
Bose is ok , just thru earphones is a problem (How to?) .

**Comment by bendover22** [comment]
What OS / version?
Don't use Windows, but in many Linux distros, playing thru headphones is kinda automatic.
Choosing the correct "output device" in the OS sound config app may be key.
May be different in later Win versions, but in 'Nix if speakers are working, then usually plugged in headphones also work.

The Win OS vol mgr app may have separate settings for headphone.
Depending on system audio mgmt app used, plugging in headphones may mute spkrs.

In 'Nix, headphone vol is controlled by: 1. the set max sys vol, 2. the current sys output vol (often an icon in systray to chg current vol level).  In 'Nix, that same systray vol level (a % of max sys vol, say 20%) affects both spkr & headphone vol.
